app/scripts/app.js

```diff
@@ -53,14 +54,19 @@ app.config(function($routeProvider) {
       templateUrl: 'views/confirm-order.html',
       controller: 'OrderController'
     })
+    .when('/payment/:id', {
+      templateUrl: 'views/payment.html',
+      controller: 'PaymentController'
+    })
     .otherwise({
       redirectTo: '/'
     });
 });
 
 app.constant('CONFIG', {
-  // 'BASE_URI': '//localhost:3000',
-  'BASE_URI': '//tma-develop.herokuapp.com',
+  'BASE_URI': '//localhost:3000',
```

Comment:
Okay, so should we try to use some env variable for setting the `BASE_URI` to kick in if the code is deployed?